bin/ChakraCore/CMakeLists.txt

```diff
@@ -75,3 +75,6 @@ if(NOT CC_XCODE_PROJECT)
     ${CHAKRACORE_BINARY_DIR}/
     )
 endif()
+
+add_dependencies(ChakraCore ChakraCoreHeaders)
```

Comment:
We better add it here now and if we another PR will change that later, that is something else.